docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -246,6 +249,23 @@ The path to the testing application. The relative file location is `{project-roo
 ```
 ---
 
+### `otherApps`
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+List of paths to apps to be pre-installed for running tests. The relative file location is `{project-root}/apps/app1.apk`, and the property supports expaned environment vairables to designate the path, as shown in the following examples.
```

Comment:
```suggestion
Set of one or more paths to apps to be pre-installed for running tests. The relative file location is `{project-root}/apps/app1.apk`, and the property supports expanded environment variables to designate the path, as shown in the following examples.
```

Reply:
ok!

Reply:
Actually the customer is supposed to not only be able to pass in filepaths inside the `otherApps` capability, but also AppStorage URIs directly. (i.e. strings prefixed with `storage:` this is so we have feature parity to appium.

I am sorry if this was not clear from the tickets, it is mentioned though on https://saucedev.atlassian.net/browse/DEVX-1064